src/components/MobX-UI/ViewBlog.tsx

```diff
@@ -0,0 +1,49 @@
+import { BlogModel, BlogView } from "../../MobX/BlogStore";
+import { Instance } from "mobx-state-tree";
+import { Button, TextInputField } from "@surya-digital/leo-reactjs-material-ui";
+import { observer } from "mobx-react";
+
+const ViewBlog = observer(() => {
+  const blogData: Instance<typeof BlogModel> = BlogView.selectedBlog;
+  return (
+    <div>
+      <div style={{ margin: "32px" }}>
+        <h1 style={{ textAlign: "center" }}>{blogData.title}</h1>
+        <p>{blogData.body}</p>
+        <h4>Comments</h4>
+        <ul>
+          {blogData.comments.map((comment) => {
+            return <li key={comment}>{comment}</li>;
+          })}
+        </ul>
+        <div>
+          <TextInputField
+            name="adding a comment"
+            value={blogData.tempComment}
+            type="text"
+            onTextChange={(value) => {
+              blogData.setTempComment(value);
+            }}
+            label="New comment..."
```

Comment:
These should be coming from i18n. All user-facing strings should be present in i18n